client/components/sorted-grid/style.scss

```diff
@@ -8,7 +8,7 @@
 .sorted-grid__label {
 	box-sizing: border-box;
 	display: flex;
-	height: 30px;
+	min-height: 30px;
```

Comment:
I think you can omit the `min-height` property completely. The min height of the element is already determined by the `line-height` and `padding` styles. Removing `min-height` locally doesn't break the UI for me.

Reply:
I've removed the fixes  in the description and min-height in the scss.